src/signals/incident-management/containers/FilterTagList/FilterTagList.js

```diff
@@ -96,23 +96,7 @@ const renderTag = (key, mainCategories, list) => {
 export const FilterTagListComponent = props => {
   const { tags, mainCategories, subCategories, directingDepartments, routingDepartments } = props;
   const { sources } = useContext(AppContext);
-  const { districts, users } = useContext(IncidentManagementContext);
-
-  const userOptions = useMemo(
```

Comment:
@boris-arkenaar I first removed this code to understand what it happening here, but couldn't figure it out. Can you explain what it does?